### PR TITLE
[draft] change meeting pace to 2 meetings for 1 PR monthly

### PR DESCRIPTION
**Pull request by glozow**
Planning to switch to monthly starting **August 2nd**.

Implements #684.

### CODE REVIEW

File: hosting.md [hunk 1]
```diff
@@ -63,6 +61,12 @@ title: Hosting a Review Club Meeting
   - Potential bugs reviewers might want to look out for.
   - Fun C++ things.
 
+- You may way to think about how you want to split the discussion across two
```
Comment (stickies-v) [inline hosting.md:64]:
```suggestion
- You may want to think about how you want to split the discussion across two
```

File: hosting.md [hunk 2]
```diff
@@ -12,26 +12,25 @@ title: Hosting a Review Club Meeting
...
   then it might be fine to discuss 100+ lines of code change. If the PR seems too
-  large to cover in a meeting, consider only discussing a subset of the commits,
-  or breaking up the discussion over multiple meetings.
+  large, consider only discussing a subset of the commits.
 
-- It's fine to discuss a PR that you opened yourself. After all, you probably
-  know more about it than anyone else!
+- In the past, we have broken up discussion on a PR into multiple meetings, and
+  run additional meetings during the week to accommodate higher interest on more
+  complex PRs.  If you feel additional time is appropriate, don't hesitate to
```
Comment (stickies-v) [inline hosting.md:32]:
```suggestion
  complex PRs. If you feel additional time is appropriate, don't hesitate to
```

File: index.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
 title: home
 ---
-### A weekly review club for Bitcoin Core PRs
+### A monthly review club for Bitcoin Core PRs
 
-<span class="question">What is this?</span> &nbsp;A weekly club for reviewing
-Bitcoin Core PRs at **{{ site.meeting_time }} on {{ site.meeting_day }}s** in
-{{ site.meeting_location }}.
+<span class="question">What is this?</span> &nbsp;A monthly club for reviewing
+Bitcoin Core PRs in {{ site.meeting_location }}.  Meetings take place on the **first
```
Comment (stickies-v) [inline index.md:8]:
```suggestion
Bitcoin Core PRs in {{ site.meeting_location }}. Meetings take place on the **first 
```

**Review by stickies-v — COMMENTED** [review]
Approach ACK, sorry for the late review.

A couple of weekly references still there (diff below) and I left a few nits because why not, but overall I think this works well. We'll probably just have to run this for a few months and update the documentation based on experience wrt format etc.

Remaining weekly references:
<details>
<summary>git diff</summary>
```diff
diff --git a/code-of-conduct.md b/code-of-conduct.md
index 5a00ac6..ef07b4f 100644
--- a/code-of-conduct.md
+++ b/code-of-conduct.md
@@ -68,7 +68,7 @@ Harrassment can take many forms. Examples include, but are not limited to:
    take further action. These actions could include:
 
     - A warning
-    - Removal from the club’s channel for the duration of that week’s meeting  
+    - Removal from the club’s channel for the duration of the ongoing meeting
     - Being banned from all future club meetings 
 
 We reserve the right to reject any report we believe to have been made in bad
diff --git a/your-first-meeting.md b/your-first-meeting.md
index 5c05768..fcb06fb 100644
--- a/your-first-meeting.md
+++ b/your-first-meeting.md
@@ -9,7 +9,7 @@ If you’re thinking about attending PR Review Club, welcome! We love new
 participants. Below is information to help you get started as a first-timer.
 
 The club meets on {{ site.meeting_day }}s at {{ site.meeting_time }} in
-{{ site.meeting_location }}. Every week, a Bitcoin protocol
+{{ site.meeting_location }}. Every month, a Bitcoin protocol
 developer will host a 60-minute discussion on a Bitcoin Core pull request. [The
 code of conduct](https://bitcoincore.reviews/code-of-conduct) details the
 behavior we expect from all participants.
@@ -40,13 +40,13 @@ very casual.
 If you’d like to more actively participate, you should clone the [Bitcoin
 repository](https://github.com/bitcoin/bitcoin). You should also check out and
 build the PR branch locally and run all tests. Take time to review the code changes and
-read the comments on that week’s PR.
+read the comments on the PR being covered.
 
 There are also notes and questions you can review and prepare. Make a note
 of any questions you want to ask.
 
 The meeting will begin with **#startmeeting** and
-“hi” from that week’s host. Feel free to respond with “hi” so people know
+“hi” from the meeting host. Feel free to respond with “hi” so people know
 you’re at your keyboard and participating. The host will poll everyone on
 whether they've reviewed the PR (y/n). Don't feel embarrassed to say "n".
 
@@ -57,6 +57,6 @@ at the end of the hour. Thanking the host at the end of the meeting is always
 appreciated!
 
 Follow the [PR Review Club on Twitter](https://twitter.com/BitcoinCorePRs) for
-announcements on that week’s PR or schedule changes.
+announcements on the next PR or schedule changes.
 
 **We look forward to having you at the next PR Review Club!**

```
</details>

**Review by glozow — COMMENTED** [review]
Thanks a bunch @stickies-v, I totally forgot about a few pages! Addressed comments and added you as coauthor.

**Review by stickies-v — APPROVED** [review]
LGTM, thanks a lot for getting this ready @glozow! (just force-pushed a tiny typo and then moved it to the correct commit in another force push, because why do things right from the start if you can avoid it)

**Pull request merged**